switchbot/adv_parser.py

```diff
@@ -93,7 +99,6 @@ class SwitchbotSupportedType(TypedDict):
         "modelName": SwitchbotModel.PLUG_MINI,
         "modelFriendlyName": "Plug Mini",
         "func": process_woplugmini,
-        "manufacturer_data_length": 12,
```

Comment:
The plug has the same length as the iometer so we had to drop this to get detection working